run_dir/static/js/projects_components.js

```diff
@@ -836,6 +838,17 @@ export const vProjectCards = {
                 }
             }
         },
+        person_assigned_name(identifier) {
+            // check that it's not undefined
+            if (! (identifier in this.$root.all_users)) {
+                return identifier
+            }
+            if (this.$root.all_users[identifier]['name'] != '') {
+                return this.$root.all_users[identifier]['name']
+            } else {
+                return identifier
+            }
```

Comment:
Would this be equivalent to the below? Not sure what you were checking for in the first if
```suggestion
            if ((identifier in this.$root.all_users) && this.$root.all_users[identifier]['name'] != '') {
                return this.$root.all_users[identifier]['name']
            } 
             return identifier
```